proseco/guide_report.py

```diff
@@ -23,7 +23,7 @@
 # Do reporting for at-most MAX_CAND candidates
 MAX_CAND = 50
 COLS = ['id', 'stage', 'forced',
-        'mag', 'mag_err', 'MAG_ACA', 'MAG_ACA_ERR',
+        'mag', 'mag_err',
```

Comment:
MAG_ACA_ERR is used directly in the spoiler tests.    I thought it was somewhat useful to have it here because I found myself looking it up for the cases where it is not presently reasonable (bright candidates).  But I suppose if the ids are links to the kadi star lookup that's fine.